File: Matrizes/ExercicioW.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Algoritmos.Matrizes
{
    internal class ExercicioW
    {
        // Ler duas matrizes A e B de 10 elementos do tipo inteiro.
        // Gerar uma nova matriz C de mesma dimensão de A e B.
        // Cada elemento de C é igual ao quadrado da soma de A e B. No final imprima os elementos da matriz C.
        static void Main(string[] args)
        {
            int[] a = new int[10];
            int[] b = new int[10];
            int[] c = new int[10];

            for (int i = 0; i < a.Length; i++)
            {
                Console.WriteLine($"Insira um número inteiro na matriz A [{i + 1} de {a.Length}]: ");
                a[i] = int.Parse(Console.ReadLine());
            }
            for (int i = 0; i < b.Length; i++)
            {
                Console.WriteLine($"Insira um número inteiro na matriz B [{i + 1} de {b.Length}]: ");
                b[i] = int.Parse(Console.ReadLine());
            }
            for (int i = 0; i < c.Length; i++)
            {
                c[i] = (int)Math.Pow((a[i] + b[i]), 2);
            }

            Console.Write("Matriz C: ");
            foreach (var item in c) Console.Write(item + " ");
        }
    }
}

File: src/main/kotlin/capitulo6/ExercicioW.kt
package capitulo6

/*
    Elaborar um programa que leia duas matrizes A e B de uma dimensão do tipo vetor com dez
elementos inteiros cada. Construir uma matriz C de mesmo tipo e dimensão que seja formada
pelo quadrado da soma dos elementos correspondentes nas matrizes A e B. Apresentar os
elementos da matriz C.
 */
fun main() {
    val a = MutableList<Int>(10) { 0 }
    val b = MutableList<Int>(10) { 0 }
    val c = MutableList<Int>(10) { 0 }

    for (i in 0..<a.size) {
        print("Insira um número inteiro na matriz A [${i + 1} de ${a.size}]: ")
        a[i] = readln().toInt()
    }
    for (i in 0..<b.size) {
        print("Insira um número inteiro na matriz A [${i + 1} de ${a.size}]: ")
        b[i] = readln().toInt()
    }

    for (i in 0..<c.size) {
        c[i] = (a[i] + b[i]) * (a[i] + b[i])
    }

    println("Matriz C:")
    for (item in c) print("$item ")

}